### ISSUE TITLE
Path has a trailing slash (/) when looking up version

### ISSUE THREAD

**Issue by rchui** [Netflix/metaflow#483]
In `metaflow==2.2.9` there is a regression where `ping/` (with a trailing slash) is appended to the url instead of `ping` in https://github.com/Netflix/metaflow/blob/f2dbb04688c3c7e9ba1ddad817f18b780982703a/metaflow/plugins/metadata/service.py#L263

The regression was introduced in this commit which went in recenthly: https://github.com/Netflix/metaflow/commit/601f7d1fbe1eaa6364a6de175b07adac2ada6f7e

This causes an error when the client attempts to ping the metadata server:
```
Metaflow 2.2.9 executing Hello for user:ryanc
Validating your flow...
    The graph looks good!
Running pylint...
    Pylint is happy!
Deploying hello-ryanc_Hello to AWS Step Functions...
    Metaflow service error:
    Metadata request (ping/) failed (code 404): 404: Not Found
```

When you attempt to curl the same endpoint the trailing slash results in an error:
```
.venv ❯ curl  https://metadata.corp.net/ping
pong
.venv ❯ curl  https://metadata.corp.net/ping/
404: Not Found
```

**Comment by savingoyal** [comment]
Merged #484. Will make a release momentarily.

**Comment by savingoyal** [comment]
release [2.2.10](https://github.com/Netflix/metaflow/actions/runs/775621969) is in progress
